lib/translators/json.js

```diff
@@ -158,23 +158,33 @@ function getKeyFilter(keys, ignored) {
   };
   keys = lodash.map(keys, stripSiblingtests);
 
-  reYes = '(' + keys.join('|') + ')';
+  var createTokenizedPattern = function(keys){
+    var re = '(' + keys.join('|') + ')';
 
-  // using a bullet char to avoid conflict with keys with periods
-  reYes = reYes.replace(/\./g, '•');
+    // tokenize escaped dots to avoid confusing them with nested dot notation
+    re = re.replace(/\\\./g, '‣');
```

Comment:
I don't see a way to do lookback assertion in javascript? That was my first instinct. Is there a way to do it?